starboard/common/media.cc

```diff
@@ -165,8 +165,8 @@ bool ParseAv1Info(std::string codec,
   // "0.110.01.01.01.0".
   // All fields are fixed size and leading zero cannot be omitted, so the
   // expected sizes are known.
-  const char kShortFormReference[] = "av01.0.05M.08";
-  const char kLongFormReference[] = "av01.0.04M.10.0.110.09.16.09.0";
+  const char kShortFormReference[] = "av01.0.05m.08";
```

Comment:
I believe there's a convention to convert the mime string to lower case in Chromium. It would be vulnerable if we go the opposite and reject mime string in lower case.

Reply:
Can you elaborate on why it's vulnerable if we pass the mime string to Starboard without converting it to lower case in Chrobalt?

Reply:
Discussed offline and modified Chromium code to avoid sending mime string in lower case.